File: api-service/src/main/java/br/com/api/model/PedidoItem.java
package br.com.api.model;

import java.math.BigDecimal;

public class PedidoItem {

    private String idItem;
    private String nomeItem;
    private String codigoItem;
    private BigDecimal valorUnitario;

    public String getIdItem() {
        return idItem;
    }

    public void setIdItem(String idItem) {
        this.idItem = idItem;
    }

    public String getNomeItem() {
        return nomeItem;
    }

    public void setNomeItem(String nomeItem) {
        this.nomeItem = nomeItem;
    }

    public String getCodigoItem() {
        return codigoItem;
    }

    public void setCodigoItem(String codigoItem) {
        this.codigoItem = codigoItem;
    }

    public BigDecimal getValorUnitario() {
        return valorUnitario;
    }

    public void setValorUnitario(BigDecimal valorUnitario) {
        this.valorUnitario = valorUnitario;
    }
}

File: front-end-maxima/src/app/model/PedidoItem.ts
export class PedidoItem {

    idItem: string;
    nomeItem: string;
    codigoItem: string;
    quantidade: number;
    valorUnitario: number;

    constructor(idItem: string,
        nomeItem: string,
        codigoItem: string,
        quantidade: number,
        valorUnitario: number) {
            this.idItem = idItem;
            this.nomeItem = nomeItem;
            this.codigoItem = codigoItem;
            this.quantidade = quantidade;
            this.valorUnitario = valorUnitario;
    }
}